superset/views/core.py

```diff
@@ -1064,6 +1064,16 @@ def save_dash(  # pylint: disable=no-self-use
         dash = session.query(Dashboard).get(dashboard_id)
         check_ownership(dash, raise_if_false=True)
         data = json.loads(request.form["data"])
+        remote_last_modified_time = data.get("last_modified_time")
+        current_last_modified_time = dash.changed_on.replace(microsecond=0).timestamp()
+        # prevent mid-air collisions
+        if remote_last_modified_time < current_last_modified_time:
+            return json_error_response(
+                "This dashboard was changed recently. "
+                "Please reload dashboard to get latest version.",
```

Comment:
wrapping in `t(..)`?

Reply:
t(..) function is JS front-end. but i added translation function for this message.